sleap_io/model/skeleton.py

```
@@ -329,3 +329,58 @@ def add_symmetry(
+            >>> skel.rename_nodes({"A": "X", "B": "Y", "C": "Z"})
+            >>> skel.node_names
+            ["X", "Y", "Z"]
+            >>> skel.rename_nodes(["a", "b", "c"])
+            >>> skel.node_names
+            ["a", "b", "c"]
+
+        Raises:
+            ValueError: If the new node names exist in the skeleton or if the old node
+                names are not found in the skeleton.
+        """
+        if type(name_map) == list:
```

Ruff

Use `is` and `is not` for type comparisons, or `isinstance()` for isinstance checks

(E721)